test/bats-unit-test/run-bats-Unit-tests.sh

```
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+sudo apt-get install -y bats &> /dev/null
```

Is this just covering both package managers and swallowing errors? 

How does one determine if there's a genuine problem getting the bats package?

---

good point. I will address by generate a log.

---

I addressed this by checking if the package existed and if not installing the packages without redirecting output